src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -29,7 +29,6 @@
         <!-- FormFactory -->
         <service id="form.factory" class="Symfony\Component\Form\FormFactory" public="true">
             <argument type="service" id="form.registry" />
-            <argument type="service" id="form.resolved_type_factory" />
```

Comment:
for 3.4?

Reply:
I think @ro0NL meant that this change could probably me made on a lower version of the FrameworkBundle too (i.e. all versions that do not support the Form component in version 3.2 or lower).